r-package/grf/R/boosted_regression_forest.R

```diff
@@ -0,0 +1,229 @@
+#' Boosted regression forest
```

Comment:
Let's add a note that this is experimental so our users aren't surprised if they're a bit slow compared to what they're used to. We could change this heading to `Boosted regression forest (experimental)` and also add a sentence saying that this functionality is experimental and will likely change in future releases.